src/getTime.js

```diff
@@ -18,7 +18,13 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const firstTime = str.match(/\b(([0-1][0-9])|(2[0-3])):[0-5][0-9]\b/g);
```

Comment:
match возвращает список совпадений, я бы назвал так
шаблон большой и чтобы его удобно изменять - вынесем в переменную
```suggestion
  const pattern = /\b(([0-1][0-9])|(2[0-3])):[0-5][0-9]\b/g;
  const matches = str.match(pattern);
```